exercises/run-length-encoding/RunLengthEncodingTest.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
```

Comment:
I think there are going to have a better flag later but for now I think having #if swift check would work.